packages/components/src/systems/RecentOrders/RecentOrdersCombineDialog.tsx

```diff
@@ -0,0 +1,204 @@
+    } catch (e) {
+      console.error('error', e);
+
+      triggerToast({
+        status: 'info',
+        header: 'Error combining orders',
+        body: 'Your order wasn’t combined, please refresh the page.'
+      });
+    }
+  };
+
+  return (
+    <Dialog
+      open={state.isCombineDialogOpen}
+      onClose={() => actions.setIsCombineDialogOpen(false)}
+      ref={ref}
+    >
+      <div class="flex flex-col gap-6">
+        <div>
+          <div class="table bg-blue-50 text-blue-600 p-2 rounded-full mb-4">
+            <Icon name="exclamationCircle" />
+          </div>
+          <Text bold>Add prescription to recent order?</Text>
+        </div>
+
+        <div class="flex flex-col gap-4">
+          <div class="flex flex-col gap-2">
+            <Text>This patient currently has an order for:</Text>
+            <div class="border border-solid border-gray-200 rounded-lg bg-gray-50 py-3 px-4">
+              <For each={fillsWithRoutingState()}>
+                {(fill) => (
+                  <div>
+                    <Text size="sm">{fill.treatment.name}</Text>
+                    <Text size="sm" color="gray">
+                      {formatRxString({
+                        dispenseQuantity: fill?.prescription?.dispenseQuantity,
+                        dispenseUnit: fill?.prescription?.dispenseUnit,
+                        fillsAllowed: fill?.prescription?.fillsAllowed,
+                        instructions: fill?.prescription?.instructions
+                      })}
+                    </Text>
+                  </div>
+                )}
+              </For>
+            </div>
+          </div>
+
+          <div class="flex flex-col gap-2">
+            <Text>
+              Select YES to combine orders and enable the patient to send it to the same pharmacy:
+            </Text>
+            <div class="border border-solid border-gray-200 rounded-lg bg-gray-50 py-3 px-4 flex flex-col gap-4">
+              <For each={state.draftPrescriptions}>
+                {(draft) => (
+                  <div>
+                    <Text size="sm">{draft.treatment.name}</Text>
+                    <Text size="sm" color="gray">
+                      {formatRxString({
+                        dispenseQuantity: draft?.dispenseQuantity ?? 0,
+                        dispenseUnit: draft?.dispenseUnit ?? '',
+                        fillsAllowed: draft?.fillsAllowed ?? 0,
+                        instructions: draft?.instructions ?? ''
+                      })}
+                    </Text>
+                  </div>
+                )}
+              </For>
+            </div>
+          </div>
+        </div>
+
+        <div class="flex flex-col items-stretch gap-2">
+          <Button
+            size="xl"
+            onClick={combineOrders}
+            disabled={isCombiningOrders()}
+            loading={isCombiningOrders()}
+          >
+            Yes, combine orders
```

Comment:
I'm assuming Jomi signed off on this? I know in the past he's preferred simple button text with explanatory text above it